server/Common/Utils/JwtUtil.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+
+namespace Common.Utils
+{
+    public class JwtUtil
+    {
+        /// <summary>
+        /// Extracts user id from jwt token
+        /// </summary>
+        /// <param name="token">JWT token generated by IdentityService</param>
+        /// <returns>User identifier</returns>
+        public static Guid GetUserIdFromToken(string token)
+        {
+            token = removeBearer(token);
+
+            var readToken = new JwtSecurityTokenHandler().ReadJwtToken(token);
+            var userId = ((List<Claim>) readToken.Claims).Find(c => c.Type.Equals("sub"));
```

Comment:
Not sure about this but I think we could go with `.First` here so we'd get a clear exception if something goes wrong